noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -203,8 +204,15 @@ impl PrivateContext {
 
     // docs:start:context_message_portal
     pub fn message_portal(&mut self, content: Field) {
```

Comment:
Not sure if it is less or more confusing that we have two functions for this. As you would just pass the `self.this_portal_address()` to send to the portal 🤷 

Buts it is a tiny thing, so we can consider, but otherwise progress.

Reply:
Fundamentally this is weird because we have the immutable portal address and now also an arbitrary recipient. Ideally we would just nuke the immutable portal address and then name the arbitrary recipient on L1 a portal.

So I feel like I would keep it as it is and create an issue for discussing whether to nuke the immutable portal and link the issue here.

But I am not going to die on this hill so I am ok with changing it.

WDYT?

Reply:
Agree, that just having one is clearer. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4512/commits/461f6d4c1d73c54e746d983a761d60c0f19e17c6